snakemake/executors.py

```diff
@@ -1719,9 +1725,8 @@ def add_workflow_files(self, job, tibanna_args):
             if src_fname != snakefile_fname:  # redundant
                 snakemake_child_fnames.append(src_fname)
         # change path for config files
-        self.workflow.overwrite_configfile, _ = self.split_filename(
-            self.workflow.overwrite_configfile, snakemake_dir
-        )
+        self.workflow.overwrite_configfiles = [self.split_filename(cf, snakemake_dir)[0]
+            for cf in self.workflow.overwrite_configfiles]
```

Comment:
Check. Sorry for not seeing this when making the change. I agree that we need to setup AWS tests ASAP.